lib/drivers/ampleforth.js

```diff
@@ -0,0 +1,49 @@
+const Driver = require('../models/driver');
+const Supply = require('../models/supply');
+
+/**
+ * Ampleforth driver. Supports total supply for native
+ * token AML.
+ *
+ * @memberof Driver
+ * @augments Driver
+ */
+class Ampleforth extends Driver {
+  constructor(options) {
+    super(
+      {
+        timeout: 100, // 10 request per second
+        supports: {
+          total: false,
+          circulating: true,
+        },
```

Comment:
I wasn't sure what to put in blockchains property because I couldn't find information about supported blockchain, is it Ampleforth or something else.